templates/data/kafka.html

```diff
@@ -0,0 +1,312 @@
+        </div>
+        <div class="col-6">
+          <p>Kafka makes for a high-performance, low latency middleware &mdash; ideal for intra-microservice communication in a modern microservices architecture, operating at scale.</p>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3">
+          <p class="p-heading--5">Continuous business intelligence</p>
+        </div>
+        <div class="col-6">
+          <p>Connect, combine and transform streaming data sources to build an up-to-the-minute view of your enterprise.</p>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row">
+        <div class="col-3">
+          <p class="p-heading--5">Data hub integration</p>
+        </div>
+        <div class="col-6">
+          <p>Integrate event and machine data into a data hub for deeper processing.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width">
+    <hr class="p-rule">
+    <div class="p-section">
+      <h2>How it works</h2>
+    </div>
+    <div class="p-section u-align--center">
+      <img src="https://assets.ubuntu.com/v1/57a2a600-kafka-diagram.svg" alt="Diagram: Canonical's automation for Apache Kafka handles deployment, configuration, security, scaling, high availability, upgrades, and monitoring. It runs on Ubuntu on cloud compute infrastructure or on Kubernetes" />
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75 is-split-on-medium">
+    <hr class="p-rule">
+    <div class="col">
+      <h2>Services</h2>
+    </div>
+    <div class="col">
+      <div class="row p-section--shallow">
+        <div class="col-3">
+          <h3 class="p-heading--5">Managed Kafka</h3>
+        </div>
+        <div class="col-6">
+          <p>Get there faster. Let us deploy and operate your Kafka data hubs, in your cloud tenancy or in your data centre.</p>
+          <hr class="p-rule is-muted">
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">Backed by an SLA</li>
+            <li class="p-list__item is-ticked">MSP and SOC2 certified service</li>
+            <li class="p-list__item is-ticked">Low cost of ownership</li>
+          </ul>
+        </div>
+      </div>
+      <hr class="p-rule">
+      <div class="row p-section--shallow">
+        <div class="col-3">
+          <h3 class="p-heading--5">Consulting</h3>
+        </div>
+        <div class="col-6">
+          <p>Expert support for your Kafka project directly from Canonical.</p>
+          <hr class="p-rule is-muted">
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">Platform architecture and design</li>
+            <li class="p-list__item is-ticked">Solution deployment</li>
+            <li class="p-list__item is-ticked">Ad-hoc consulting</li>
+          </ul>
```

Comment:
Missing datasheet? I created an asset based on the datasheet provided https://assets.ubuntu.com/v1/7c70b24d-Canonical.Managed.Kafka.datasheet_29.11.21.pdf